packages/xarc-react-router/src/browser/router-history.ts

```diff
@@ -0,0 +1,4 @@
+import { createBrowserHistory } from "history";
+export default class RouterHistory {
```

Comment:
this is a bit overkill to create a static value, and no `export default`

Reply:
also, this will trigger `createBrowserHistory` immediately when app's loaded

Reply:
Fixed